src/app/dashboard/Home/OverviewItem.jsx

```diff
@@ -26,7 +25,7 @@ const HomeItem = ({ icon, link, linkLabel, nbr, textColor, title }) => {
             <H3 style={{ color: textColor }}>{title}</H3>
```

Comment:
what textColor is this ? why dont we use theme from useStyles here ?

Reply:
sounds good to me - fwiw this is not code i wrote though.. was here before the PR.. its a property set from above - so planning is one color, in progress another etc.